### PR TITLE
Add Sigma Prime boot ENR

### PR DESCRIPTION
**Pull request by paulhauner**
Adds the Sigma Prime (Lighthouse) boot ENR. Credits to @antondlr for spinning up this node.

There was previously no newline at the end of the file so it looks like I deleted an ENR and added it again.

### CODE REVIEW

**Comment by paulhauner** [comment]
Oh no, we need to regenerate these with the new Capella fork epoch. **Please don't merge yet.**

**Comment by paulhauner** [comment]
I've updated our ENR and it's ready to publish!

```
enr-cli read enr:-Le4QI88slOwzz66Ksq8Vnz324DPb1BzSiY-WYPvnoJIl-lceW9bmSJnwDzgNbCjp5wsBigg76x4tValvGgQPxxSjrMBhGV0aDKQqX6DZjABcAAAAQAAAAAAAIJpZIJ2NIJpcIQ5gR6Wg2lwNpAgAUHQBwEQAAAAAAAAADR-iXNlY3AyNTZrMaEDPMSNdcL92uNIyCsS1
77Z6KTXlbZakQqxv3aQcWawNXeDdWRwgiMohHVkcDaCI4I
ENR Read:
Sequence No:1
NodeId: 07857f43ab2782872258a4309465ccb4ef893c29a72e417667e9044f919754fe
EnodeId: enode://07857f43ab2782872258a4309465ccb4ef893c29a72e417667e9044f919754fe@57.129.30.150?discport=9000
Libp2p PeerId: 16Uiu2HAmGkCeLewxvqo4aLp3hFmkui9TFhT7W9zHH4GXN5mgY4fg
IP:57.129.30.150
IP6:2001:41d0:701:1000::347e
UDP Port:9000
UDP6 Port:9090
Eth2 Field:
        Fork digest: a97e8366
        Next fork version: 30017000
        Next fork epoch: 256
        SSZ Bytes: a97e8366300170000001000000000000
Known multiaddrs:
/ip4/57.129.30.150/udp/9000
/ip6/2001:41d0:701:1000::347e/udp/9090
```

**Comment by paulhauner** [comment]
You might as well merge #63 before this and I'll fix the merge conflict.

File: custom_config_data/boot_enr.txt
Comment (rolfyone) [inline custom_config_data/boot_enr.txt:5]: Barnabus suggested ownership being in README, and these two files just being a long list, so i guess consistent with the teku PR to my understanding...
Based on that probably remove the comment in boot_enr for `#Sigma Prime` and same in bootstrap nodes
Then add this ENR to the README with the comment.... 3 places is a lot...

Comment (paulhauner) [reply custom_config_data/boot_enr.txt]: Makes sense, thanks. I'll follow the standard.

**Comment by paulhauner** [comment]
This is ready for review!

File: README.md
Comment (jimmygchen) [inline README.md:71]: Should probably move this above "Deposit contract address"

Comment (paulhauner) [reply README.md:70]: Done, thanks

**Review by barnabasbusa — APPROVED** [review]

**Pull request merged**
